### DISCUSSION TITLE
Failed to find a valid digest in the 'integrity' attribute for resource

### DISCUSSION THREAD

**Discussion by kaovilai** [vercel/community#2100] [Bug Report]
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/11228024/232383203-3c317c13-aa0a-4590-8222-b0c06628d227.png">
This looks broken on my domain name

[velero.tig.pw](http://velero.tig.pw/)

Domain config
```
CNAME | velero | cname.vercel-dns.com
```
Built from repo https://github.com/kaovilai/velero

Looks fine on vercel owned domains.
[velero-git-main-kaovilai.vercel.app](https://velero-git-main-kaovilai.vercel.app/)
[velero-rbmmtu01a-kaovilai.vercel.app](https://velero-rbmmtu01a-kaovilai.vercel.app/)

Similar issue seen in stackoverflow
[Failed to find a valid digest in the 'integrity' attribute for resource 'https](https://stackoverflow.com/questions/69926878/failed-to-find-a-valid-digest-in-the-integrity-attribute-for-resource-in-blazo)

```
 the 'integrity' attribute for resource 'https://velero.tig.pw/styles.666a5dc1cd80e47092a71103358e46b56345de6f3044680f856f219cbc2fe91b.css' with computed SHA-256 integrity 'u3fvXrmLoENiM99fWiOcD7X8XN6BQ1hyFZ4Sitm1keA='. The resource has been blocked.
```

**Comment by wiiiimm** [comment]
![image](https://user-images.githubusercontent.com/179761/232682705-5bdb1183-d27a-440e-bae1-dc89979f69f8.png)

Your site looks fine on my browser. Can you try clearing cache or try to use another browser / device?

**Reply by kaovilai** [reply]
Weird.. it didn't work in safari+edge+mobile yesterday

**Reply by kaovilai** [reply]
It's still happening actually on the same device.. let me try another device

**Reply by kaovilai** [reply]
Ok.. so it's not cookies.. disabling cache in Browser Inspect resolved the issue.

**Reply by kaovilai** [reply]
Safari still having this issue.. so I guess I need to clear cache there as well.

**Comment by kaovilai** [comment]
Thanks @williamli ! ++
